mo_net/model/model.py

```
@@ -3,25 +3,26 @@
 import pickle
 from collections.abc import Callable, Mapping, MutableSequence
 from dataclasses import dataclass
-from functools import partial, reduce
+from functools import partial
 from itertools import chain
 from operator import itemgetter
 from pathlib import Path
 from typing import (
     IO,
+    Callable,
```

Ruff (0.11.9)

Redefinition of unused `Callable` from line 4

(F811)